base/complex.jl

```diff
@@ -524,6 +630,12 @@ function log1p{T}(z::Complex{T})
     end
 end
 
+"""
+    ^(x, y)
+
+Exponentiation operator.
+"""
+Base.:(^)(x, y)
```

Comment:
This was a fantastic explanation! Thanks a lot. I updated accordingly and am no longer getting any warnings from Documenter.